common/src/main/kotlin/no/nav/su/se/bakover/common/Config.kt

```diff
@@ -543,14 +543,21 @@ data class ApplicationConfig(
             clientsConfig = ClientsConfig.createFromEnvironmentVariables(),
             kafkaConfig = KafkaConfig.createFromEnvironmentVariables(),
             unleash = UnleashConfig.createFromEnvironmentVariables(),
-            jobConfig = JobConfig(
-                personhendelse = JobConfig.Personhendelse(naisCluster()),
-                konsistensavstemming = when (naisCluster()) {
-                    NaisCluster.Dev -> JobConfig.Konsistensavstemming.Dev()
-                    NaisCluster.Prod -> JobConfig.Konsistensavstemming.Prod()
-                    null -> throw IllegalStateException("Kunne ikke identifsiere nais-cluster")
-                },
-            ),
+            jobConfig = naisCluster().let { cluster ->
+                if (cluster == null) throw IllegalStateException("Kunne ikke identifsiere nais-cluster")
+
+                JobConfig(
+                    personhendelse = JobConfig.Personhendelse(naisCluster()),
+                    konsistensavstemming = when (cluster) {
+                        NaisCluster.Dev -> JobConfig.Konsistensavstemming.Dev()
+                        NaisCluster.Prod -> JobConfig.Konsistensavstemming.Prod()
+                    },
+                    initialDelay = when (cluster) {
+                        NaisCluster.Dev -> 0
```

Comment:
NaisCluster.Dev tilsvarer preprod. Jeg antar de samme problemene gjelder for dev/prod-miljøene.